python/src/scipp/table_html/formatting_html.py

```diff
@@ -96,19 +92,14 @@ def retrieve(var, variances=False, single=False):
 
 def _short_data_repr_html_non_sparse(var, variances=False):
     if hasattr(var, "data"):
-        data_repr = repr(retrieve(var.data, variances))
+        return repr(retrieve(var.data, variances))
     else:
-        data_repr = repr(retrieve(var, variances))
-    return data_repr
+        return repr(retrieve(var, variances))
 
 
 def _short_data_repr_html_sparse(var, variances=False):
-    s = []
-    non_sparse_dim = var.dims[0]
-    for i in range(var.shape[0]):
-        s.append(repr(retrieve(var[non_sparse_dim, i], variances)))
-
-    return "\n".join(s)
+    return "array([" + ",\n       ".join(
```

Comment:
I think this adds a lot of noise
![image](https://user-images.githubusercontent.com/9135965/70791359-e4d5cd80-1d8e-11ea-93b0-3bb2360ac4d6.png)
I don't think `list(` should be shown in the preview, only in the details.

Reply:
I think the bigger discussion is whether we should have the multi-line preview at all? It takes a lot of vertical space.

Reply:
To elaborate why I put the `list` everywhere: Without it, the display is indistinguishable from a dense 2D array, as soon as there are 5 or more entries in every list.

Reply:
If we don't have it for sparse values, I'm not sure what else should be shown that is not misleading. I don't think simply showing the first and last two number is good at all in sparse data

Reply:
Maybe we should show the sizes?

Reply:
I think it might be the best approach. I couldn't really come up with a format to display it, `0: 3, 1: 5` or `0 - 3; 1 - 5` aren't immediately clear

Reply:
`sparse(len=3), sparse(len=5)`?

Reply:
Looks good!

Reply:
How about `sparse(len=3, len=6, ... len=10)`, so it can fit in a single line?

Reply:
Looks even better

Reply:
`sparse(len=3, ... len=10)` takes too much space, even if I show only first and last item. Maybe we can try moving variances to the next line?

In any case, I suggest we do that in a follow-up PR, this is fine-tuning now, and we should first tackly some of the other more pressing aspects.

Reply:
I a putting `len=3, ..., len=10` now, dropping the `sparse()` around it. That fits, can improve later.